### PR TITLE
add release v0.8.0

### PR DESCRIPTION
**Pull request by bxy4543**
add release v0.8.0

### CODE REVIEW

File: release_note.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+## v0.8.0
 
-*🚀 feat*:
+*🚀 feat/bugfix/optimize*:
 
-- [x] support sudo to exec command and remote command
-- [x] support render env in Clusterfile
\ No newline at end of file
+- [x] **!! support multi image, the previous version image is not available and needs to be pulled and built again**
+- [x] ssh passwd in clusterfile encrypt
```
Comment (kakaZhou719) [inline release_note.md:8]: encrypt ssh passwd in clusterfile

File: release_note.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+- [x] join also deployed plugins
```
Comment (kakaZhou719) [inline release_note.md:9]: run plugins process when execute sealer join

File: release_note.md [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+- [x] fix build cause OOM when saving images from imageList file
```
Comment (kakaZhou719) [inline release_note.md:10]: fix pulling docker image  encounter OOM  issue

File: release_note.md [hunk 4]
```diff
@@ -1,8 +1,13 @@
...
+- [x] fix build pulls the private repository image
```
Comment (kakaZhou719) [inline release_note.md:11]: add cert config for pulling private repository

**Review by Stevent-fei — APPROVED** [review]

**Pull request merged**